packages/agreement-process/open-api/agreement-service-spec.yml

```diff
@@ -804,6 +804,15 @@ paths:
               format: uuid
             default: []
           explode: false
+        - in: query
+          name: states
+          description: comma separated sequence of states
+          schema:
+            type: array
+            items:
+              $ref: "#/components/schemas/AgreementState"
+            default: []
+          explode: false
```

Comment:
For comma separated array values we have to manually patch the generated zodios decoder (see the catalog-process patchZodios.ts file)

Reply:
😨 right, the scary patch we applied in catalog 🥲 
It is really scary that this is missed both by our automated tests and by the manual tests I did...
But we already discussed that when implementing it, I tracked it here and will do it in a further PR to keep things cleaner: https://pagopa.atlassian.net/browse/IMN-416

Do you know if there were any updates on the `zodios` issue that was causing this?

Reply:
still open. The only response that I get is 
```
You can try to configure paramsSerializer of axios.
[paramsSerializer Option](https://github.com/axios/axios#request-config)
```
but I think that this config only affects the generated zodios client. I'm waiting for an answer.

Reply:
Amazing, let's hope we can solve it soon! Could you link the issue/PR here so that we have a reference also in this discussion? 🙏

Reply:
https://github.com/ecyrbe/zodios/issues/589